Makefile

```diff
@@ -104,27 +108,40 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
+.PHONY: openapi
```

Comment:
I don't think this needs to be phony. So far it works well.

Reply:
Makefile will work without naming targets as phony. Phony means only that a target is one that is not really the name of a file. More in https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html